src/period-calculation/period-types.ts

```diff
@@ -0,0 +1,90 @@
+import { PeriodIdentifier } from './types'
+
+export const FIXED_PERIOD_TYPES: { [name: string]: PeriodIdentifier } = {
+    DAILY: 'Daily',
+    WEEKLY: 'Weekly',
+    WEEKLYWED: 'WeeklyWednesday',
+    WEEKLYTHU: 'WeeklyThursday',
+    WEEKLYSAT: 'WeeklySaturday',
+    WEEKLYSUN: 'WeeklySunday',
+    BIWEEKLY: 'BiWeekly',
+    MONTHLY: 'Monthly',
+    BIMONTHLY: 'BiMonthly',
+    QUARTERLY: 'Quarterly',
+    QUARTERLYNOV: 'QuarterlyNov', // used in Ethiopi: 'SixMonthly',
+    SIXMONTHLY: 'SixMonthly',
+    SIXMONTHLYAPR: 'SixMonthlyApril',
+    SIXMONTHLYNOV: 'SixMonthlyNov', // used in Ethiopia
+    YEARLY: 'Yearly',
+    FYAPR: 'FinancialApril',
+    FYJUL: 'FinancialJuly',
+    FYOCT: 'FinancialOct',
+    FYNOV: 'FinancialNov',
+}
```

Comment:
I don't think we need this at all if we have a good `PeriodIdentifier` type (string literal)

Reply:
This completely changed, see [my comment further above](https://github.com/dhis2/multi-calendar-dates/pull/12#discussion_r1107934306).